cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,204 @@
-        <h3>MITx MicroMasters Programs</h3>
-        <ul class="current-programs-list">
-        {% for program, programpage in programs %}
-          <li class="col-md-4">
-            <div class="program-thumbnail">
-              {% if programpage %}
-                  <a href="{{ programpage.url }}" class="program-link">
-              {% endif %}
-                <div class="program-info">
-                  <h4 class="program-title" aria-describedby="program-{{ program.id }}-description">
-                    {% if programpage %}
-                      {{ programpage.title }}
-                    {% endif %}
-                  </h4>
-                  {% with program.course_set.count as course_count %}
-                    {% if course_count %}
-                      <div class="program-num-courses">
-                        {% blocktrans count counter=course_count %}
-                          {{ counter }} course
-                        {% plural %}
-                          {{ counter }} courses
-                        {% endblocktrans %}
-                      </div>
-                    {% endif %}
-                  {% endwith %}
-                </div>
-                {% if programpage and programpage.thumbnail_image %}
-                  {% image programpage.thumbnail_image fill-690x530 as thumbnail_image %}
-                  <img src="{{ thumbnail_image.url }}" alt=""
-                       >
-                {% else %}
-                  <img src="{% static 'images/course-thumbnail.png' %}"
-                       alt="" class="program-default-image">
-                {% endif %}
-              {% if programpage %}
-                </a>
-              {% endif %}
-            </div>
-            <div class="program-description">
-              <p class="program-description-text" id="program-{{ program.id }}-description">{{ program.description|default:"No description available for this program." }}</p>
-
-              {% if programpage %}
-                <p class="program-description-link">
-                  <a href="{{ programpage.url }}" class="program-link">
-                    More about {{ program }}
-                  </a>
-                </p>
-              {% endif %}
-            </div>
-          </li>
-        {% endfor %}
-
-        </ul>
-        <br clear="all">
 
-        <div class="more-coming-soon">
-          <p>MIT plans to bring more MicroMasters programs to motivated learners like you. Sign up for our mailing list to hear about new MicroMasters programs.</p>
-          <p>
-            <a data-toggle="modal" data-target="#subscribeAlert" class="sign-up-link">Sign Up for Mailing List</a>
-          </p>
+      <section class="events">
+        <div class="container">
+          <h3>Join us for one of our upcoming free events online</h3>
+          <div class="content-holder">
+            <a href="events/"><img src="{% static 'images/events.png' %}" alt="Events"></a>
```

Comment:
I'm seeing a weird red box next to the image. It seems to be part of the link:

<img width="643" alt="image" src="https://user-images.githubusercontent.com/430126/105038531-705b6c80-5a2d-11eb-8574-a492f4030719.png">

I'm seeing in Firefox, Safari and Chrome.